### PR TITLE
Openstack Security Group Edit and Delete

### PR DESCRIPTION
**Pull request by gadresushrut**
Please review my code for edit and delete functionality.

### CODE REVIEW

File: app/scripts/modules/openstack/securityGroup/details/details.controller.js [hunk 1]
```diff
@@ -1,6 +1,113 @@
-//TODO: Need to implement the code for details, when backend is ready.
 'use strict';
 
 let angular = require('angular');
 
-module.exports = angular.module('spinnaker.securityGroup.openstack.details.controller', []);
+module.exports = angular.module('spinnaker.securityGroup.aws.details.controller', [
```
Comment (anotherchrisberry) [inline app/scripts/modules/openstack/securityGroup/details/details.controller.js]: s/aws/openstack

File: app/scripts/modules/openstack/securityGroup/transformer.js [hunk 2]
```diff
@@ -26,14 +26,30 @@ module.exports = angular.module('spinnaker.openstack.securityGroup.transformer',
...
-          type: 'tcp'
+          ruleType: 'tcp'
 
       };
     }
 
+    function prepareForEdit(securityGroup) {
+
+      securityGroup.rules = _.map(securityGroup.inboundRules, function(x) {
+          return{
```
Comment (anotherchrisberry) [inline app/scripts/modules/openstack/securityGroup/transformer.js]: should be a space between `return` and `{`

**Comment by anotherchrisberry** [comment]
Aside from a couple of very small things, this LGTM. Does anyone else need to review it?

**Comment by gadresushrut** [comment]
Done addressing the review comments.

**Pull request merged**